src/utils/ucc_coll_utils.c

```diff
@@ -631,20 +631,22 @@ ucc_ep_map_t ucc_ep_map_create_reverse(ucc_rank_t size)
     return map;
 }
 
-static inline int ucc_ep_map_is_reverse(ucc_ep_map_t *map)
+static inline int ucc_ep_map_is_reverse(ucc_ep_map_t *map,
+                                        int reversed_reordered_flag)
```

Comment:
Would it be clearer to name this variable `use_reordering` ?
I'm honestly not sure, it's just that the name `reversed_reordered_flag` is not so clear to me but I let you decide what you think's best

Reply:
This flag should be true only for the case of both reordering and reverse (reverse meaning the second frag when using bidirectional), so not for just reordering.